src/apps/testapps/testCellToChildren.c

```diff
@@ -63,11 +63,17 @@ static void assertSetsEqual(H3Index *set1, int len1, H3Index *set2, int len2) {
     assertSubset(set2, len2, set1, len1);
 }
 
-static void checkChildren(H3Index h, int res, H3Index *expected,
-                          int numExpected) {
-    int64_t numChildren = H3_EXPORT(cellToChildrenSize)(h, res);
+static void checkChildren(H3Index h, int res, H3Error expectedError,
+                          H3Index *expected, int numExpected) {
+    int64_t numChildren;
+    H3Error numChildrenError =
+        H3_EXPORT(cellToChildrenSize)(h, res, &numChildren);
+    t_assert(numChildrenError == expectedError, "Expected error code");
+    if (expectedError != E_SUCCESS) {
+        return;
+    }
```

Comment:
Does this imply that there are no errors we can test that won't be returned by `cellToChildrenSize`? I.e. there are no errors in `cellToChildren` that aren't in `cellToChildrenSize`?

Reply:
`cellToChildren` always returns `E_SUCCESS` right now, so that is correct